libgalois/src/analytics/pagerank/pagerank-impl.h

```diff
@@ -31,7 +31,13 @@
 #include "katana/analytics/pagerank/pagerank.h"
 
 typedef float PRTy;
+struct ValueOut {
```

Comment:
This is a very vague name. Can you rename this something that actually specifies what it means?

Reply:
Hmmmm it is very hard to think of short and clear name :-) I decided to use `PROperandsTy` and `using PROperands = katana::PODProperty<PROperandsTy>` .

Reply:
Why not just `PagerankValueAndOutputCount` (I'm totally guessing on what `out` is, so making that clear would be useful).

Reply:
Out is an out degree. So `PagerankValueAndOutDegree` looks right to use!